src/oge/impute_hourly_profiles.py

```diff
@@ -236,7 +236,7 @@ def select_best_available_profile(hourly_profiles):
         ] = profile
         # fill missing values with non-missing values from the filtered profile data
         hourly_profiles["profile"] = hourly_profiles["profile"].fillna(
-            hourly_profiles[profile]
+            hourly_profiles[profile].astype("float64")
```

Comment:
Why is this change necessary?

Reply:
I try to fix the following warning:
```
FutureWarning: Setting an item of incompatible dtype is deprecated and will raise an error in a future version of pandas. Value 'residual_profile_filtered' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.
```
but it doesn't work so I will remove it.

Reply:
Removed